packages/ckeditor5-table/tests/tableproperties/tablepropertiesediting.js

```diff
@@ -228,6 +228,38 @@ describe( 'table properties', () => {
 					assertTRBLAttribute( table, 'borderStyle', 'solid' );
 					assertTRBLAttribute( table, 'borderWidth', '1px' );
 				} );
+
+				// https://github.com/ckeditor/ckeditor5/issues/8393.
+				it( 'should not upcast contents of nested table', () => {
+					editor.setData( '<table style="border:1px solid red">' +
+						'<tr>' +
+							'<td>parent:00</td>' +
+							'<td>' +
+								'<table style="border:1px solid green"><tr><td>child:00</td></tr></table>' +
+							'</td>' +
+						'</tr>' +
+					'</table>' );
+
+					expect( getModelData( editor.model ) ).to.equal(
```

Comment:
This way of testing stands out here, but because I wanted to focus on the contents, not the style, I went with `getModelData` here.

Reply:
This is a perfectly valid code comment :) You can add it inline.